mathics/core/symbols.py

```diff
@@ -196,18 +197,18 @@ def __cmp(self, other) -> int:
 
 class Atom(BaseElement):
     """
-    Atoms are the leaves (in the common tree sense, not the Mathics
-    ``_elements`` sense) and Heads of an Expression or M-Expression.
+    Atoms are the (some) leaves and the Heads of an S-Expression or an M-Expression.
 
-    In other words, they are the units of an expression that we cannot
-    dig down deeper structurally.  Various object primitives i.e.
-    ``ByteArray``, `CompiledCode`` or ``Image`` are atoms.
+    In other words, they are the some of units of an expression that we cannot
+    dig down deeper structurally.
 
     Of note is the fact that the Mathics ``Part[]`` function of an
     Atom object does not exist.
 
     Atom is not a directly-mentioned WL entity, although conceptually
     it very much seems to exist.
+
+    There are kind of leaf note is the Builtin, i.e. `ByteArray``, `CompiledCode`` or ``Image``.
```

Comment:
I do not get this sentence.

Reply:
Yes, this is a mangled sentence. Is this clearer: 

    The other kinds expression leaf is a ``Builtin``, e.g. `ByteArray``, `CompiledCode`` or ``Image``.

Reply:
OK, better :)